server/controllers/api/userCtrl.js

```diff
@@ -23,11 +23,12 @@ async function create(req, res) {
   try {
     const user = await User.create(req.body)
     const token = createJWT(user)
-    //res.json(token) -> previous code
+    // res.json(token)
     //insert token storage into a cookie here.
     res.json(user);
+    console.log('User created:', user)
   } catch(error) {
-    res.status(400).json(error)
```

Comment:
Here can you add a console.log for the catch block as well? It'll be helpful for the developers to know if a signup failed due to a duplicate email as we continue to test and refine these features out.